Utilities.Tests/Extensions/DictionaryExtensionsTests.cs

```diff
@@ -29,8 +29,8 @@ public void AddMany_ShouldAddAllKeyValuePairs()
 
         dictionary.Should().HaveCount(3);
 
-        dictionary.Should().ContainKey("one").WhichValue.Should().Be(1);
-        dictionary.Should().ContainKey("two").WhichValue.Should().Be(2);
-        dictionary.Should().ContainKey("three").WhichValue.Should().Be(3);
+        dictionary.Should().ContainKey("one");
+        dictionary.Should().ContainKey("two");
+        dictionary.Should().ContainKey("three");
```

Comment:
Could you also check values for these keys as it was before?

Reply:
Thanks for the inputs. WhichValue is not working post update so used WhoseValue instead which should do the same thing.